fuzz/fuzz_rfc2045.ml

```diff
@@ -0,0 +1,52 @@
+module Rfc2045 = struct
+  exception Encode_error of string
+  exception Decode_error
+
+  let check_encode str =
+    let subs = Astring.String.cuts ~sep:"\r\n" str in
+    let check str =
+      if String.length str > 78
+      then
+        raise (Encode_error "too long string returned")
+    in List.iter check subs; str
+
+  let encode ?pad:_ ?alphabet:_ input =
+    let buf = Buffer.create 80 in
+    let encoder = Rfc2045.encoder (`Buffer buf) in
+    String.iter
+      (fun c ->
+        let ret = Rfc2045.encode encoder (`Char c) in
+        match ret with
+          | `Ok -> ()
+          | _ -> raise (Encode_error "encoding was not Ok"))
+      input;
+    let encode = Rfc2045.encode encoder (`End) in
+    match encode with
+      | `Ok -> Buffer.contents buf |> check_encode
+      | _ -> raise (Encode_error "encoding was not Ok")
+
+  let decode ?alphabet:_ input =
+    let decoder = Rfc2045.decoder (`String input) in
+    let rec rec_decode acc =
+      match Rfc2045.decode decoder with
+        | `End -> acc
+        | `Flush output -> rec_decode (Bytes.(of_string output |> copy |> to_string)::acc)
+        | _ -> raise Decode_error
```

Comment:
Just a note (but the code is fine), it's better to write all cases than `_`. In this context for example, as before, `Await` will never appear (because it was made by `String`). However, `Malformed` can appear, and, in this branch, indeed, we need to `raise` an error.